hidimstat/standardized_svr.py

```diff
@@ -5,30 +5,48 @@
 from sklearn.pipeline import Pipeline
 
 
-def standardized_svr(X, y, Cs=np.logspace(-7, 1, 9), return_C=False, n_jobs=1):
+def standardized_svr(X, y, Cs=np.logspace(-7, 1, 9), n_jobs=1):
+    """Cross-validated SVR
+
+    Parameters
+    -----------
+    X : ndarray or scipy.sparse matrix, shape (n_samples, n_features)
+        Data.
+
+    y : ndarray, shape (n_samples,) or (n_samples, n_targets)
+        Target. Will be cast to X's dtype if necessary.
+
+    Cs : ndarray, optional (default=np.logspace(-7, 1, 9))
+        The linear SVR regularization parameter is set by cross-val running
+        a grid search on the list of hyper-parameters contained in Cs.
+
+    n_jobs : int or None, optional (default=1)
+        Number of CPUs to use during the cross validation.
+
+    Returns
+    -------
+    beta_hat : array, shape (n_features,)
+        Estimated parameter vector.
+
+    scale : ndarray, shape (n_features,)
+        Value of the standard deviation of the parameters.
+    """
 
     n_samples, n_features = X.shape
 
-    steps = [('SVR', LinearSVR())]
+    if Cs is None:
```

Comment:
You probably don't need that, since there's a default Cs. Otherwise you should test it